packages/xod-client/src/workers/run.js

```diff
@@ -0,0 +1,27 @@
+import * as R from 'ramda';
+import { noop } from 'xod-func-tools';
+import { prepareModuleOptions } from 'xod-cloud-tabtest';
+
+import WasmWorker from './wasm.worker';
+
+const runWasmWorker = ({ suite, runtimeUrl, handlers = {} }) => {
+  const worker = new WasmWorker();
+  const workerId = Date.now(); // TODO: Replace it!
+  worker.id = workerId;
```

Comment:
If `id` isn’t a native property of a worker, should we inject it here? It’s better to return a record with `id` and `worker` as fields. Also, should the `Date.now` be replaced with a Posix timestamp or something?